sound/soc/sof/imx/imx8m.c

```diff
@@ -204,6 +235,12 @@ static int imx8m_probe(struct snd_sof_dev *sdev)
 		goto exit_pdev_unregister;
 	}
 
+	priv->dap = devm_ioremap(sdev->dev, IMX8M_DAP_DEBUG, IMX8M_DAP_DEBUG_SIZE);
+	if (!priv->dap) {
+		dev_err(sdev->dev, "error: failed to map DAP debug memory area");
+		return -ENODEV;
```

Comment:
error handling is inconsistent with blocks above. Should this be:

ret = -ENODEV;
goto exit_pdev_unregister;

??